src/components/MermaidDash/MermaidDash.jsx

```diff
@@ -204,7 +205,17 @@ const MermaidDash = () => {
         ) : null}
 
         <DesktopToggleFilterPaneButton onClick={handleShowFilterPane}>
-          {showFilterPane ? String.fromCharCode(10094) : String.fromCharCode(10095)}{' '}
+          {showFilterPane ? (
+            <>
+              <StyledChevronSpan>{String.fromCharCode(10094)}</StyledChevronSpan>
+              <span>Filters</span>
+            </>
+          ) : (
+            <>
+              <StyledChevronSpan>{String.fromCharCode(10095)}</StyledChevronSpan>
+              <span>Filters</span>
```

Comment:
`<span>Filters</span>` is repeated. Can it be put outside of the ternary instead?